src/renderer/config-blocks/SettingsButton.svelte

```diff
@@ -75,28 +94,100 @@
   ];
 
   let suggestionElement = undefined;
+  let minMaxEnabled = false;
+
+  function calculateStepValues(steps, min, max) {
```

Comment:
Example: 3 step mode, min=-10 max=20 displayed steps are incorrectly: 0, 10, 20, 30 (should be -10, 0, 10, 20)

first step should always be the min value, then each step is calculated from that